api/cfm-openapi.yaml

```diff
@@ -49,6 +49,34 @@ paths:
               schema:
                 $ref: "#/components/schemas/serviceInformation"
 
+  /cfm/v1/discover:
+    get:
+      description: Discover devices on the network.
+      operationId: discoverDevices
+      parameters:
+        - name: type
+          in: query
+          description: Filter devices by type (either 'cma' or 'cxl-host')
+          required: true
+          schema:
+            type: string
+            enum: [cma, cxl-host]
```

Comment:
The following change is related to several other changes in the PR.
Please look through all my comments to fully understand the changes.

**_Replace_** 61-63 with:
```
schema:
     $ref: '#/components/schemas/discoveredDevice```

Reply:
Here is the input parameter deviceType(`enum: [cma, cxl-host]`), so I added a parameter `deviceType` under `components`.